### DISCUSSION TITLE
on successfull redirection from keycloak to superset ,user is not getting auto logged in

### DISCUSSION THREAD

**Discussion by shashi5678** [apache/superset#18775] [Q&A / Help] [answered]
I am trying to integrate Keycloak SSO integration with Superset ,

On click a link from Superset login screen , I am able to reach keycloak application for authentication and on successful login , the keycloak application is successfully redirecting me to Superset login screen but the user is not getting logged in automatically. Any help on this would be highly appreciated

Redirection URL at Superset end from keycloak

http://localhost:8088/login/#session_state=19371d8f-1bd6-41cb-840f-2b11ba477a30&code=63c8f86f-146f-41cd-a2b0-8fb70872b70b.19371d8f-1bd6-41cb-840f-2b11ba477a30.c83b9a3c-f556-446c-99ca-fb56cad1c7d2

I have also creates a test user on both superset and keycloak level

**Comment by zhaoyongjie** [comment, marked as answer]
Hey @shashi5678, Please fill out the full of issue template. 
- What is Keycloak
- Which authentication do you use in Superset

**Reply by shashi5678** [reply]
I am trying to integrate Keycloak SSO integration with Superset ,
On click a link from Superset login screen , I am able to reach keycloak  application for authentication and on successful login , the keycloak application is successfully redirecting me to Superset login screen but the user is not getting logged in  automatically.  Any help on this would be highly  appreciated

**Reply by musicmuthu** [reply]
@shashi5678  Can you share the configuration details
